# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ooo/bdb43.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- /dev/null	1969-12-31 21:00:00.000000000 -0300
+++ ./patches/src680/bdb43.diff	2005-05-07 11:36:54.000000000 -0400
@@ -0,0 +1,12 @@
+--- config_office/configure.in~	2005-05-07 11:33:14.424136528 -0400
++++ config_office/configure.in	2005-05-07 11:34:44.783696512 -0400
+@@ -2448,8 +2448,7 @@
+         ], []
+     )
+     AC_MSG_CHECKING([whether db is at least 4.1])
+-    # FIXME: Will db 4.3 work? Can't test it since the 4.3 java bindings are not packaged :/
+-    for v in 1 2; do
++    for v in 1 2 3; do
+        AC_TRY_RUN([
+ #include <db.h>
+ 
